### PR TITLE
[feature] travis wheelhouse

### PR DESCRIPTION
**Pull request by icereval**
Purpose
---

Speed up travis-ci builds.

Changes
---

Added WHEELHOUSE environment variable.
Added wheelhouse invoke task to download the wheel repo to the path specified.
Modified invoke requirements, addon_requirements and mfr to reference the wheel path.

Side effects
---

If travis-ci changes their server builds or we update requirements, we'll need to build the corresponding whl files and update our repository.

Worst case scenario the build time should simply increase until we update the wheelhouse repo.

### CODE REVIEW

**Comment by lbanner** [comment]
Fast! @icereval :smiling_imp: TY

File: requirements.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ elasticsearch==1.3.0
 # Development version of modular-odm
 git+https://github.com/CenterForOpenScience/modular-odm.git@develop
 # Development version of modular file renderer
-git+https://github.com/CenterForOpenScience/modular-file-renderer.git@dev
+git+https://github.com/icereval/modular-file-renderer.git@dev
```
Comment (icereval) [inline requirements.txt]: REMINDER: This needs to be set back to COS before commit.

**Comment by icereval** [comment]
##### Times from the following commands:

`travis_retry invoke requirements`
`travis_retry invoke addon_requirements`
`travis_retry mfr -r install all`

*We can likely add additional dependancies to our wheelhouse to speed these up further*

Before
----

image


After
---

image

**Comment by sloria** [comment]
:ship: :it: Merged in https://github.com/CenterForOpenScience/osf.io/commit/071bda9d94849d7cdbf701a9c54a9fd644df6a98

**Pull request closed without merging**